package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "cuttle",
   "description": "The deepest card game under the sea",
-  "version": "4.4.2",
+  "version": "4.4.3",
```

Let's bump the version again
```suggestion
  "version": "4.4.4",
```